autoload/leaderf/python/leaderf/fileExpl.py

```diff
@@ -28,6 +29,26 @@ def deco(*args, **kwargs):
             return func(*args, **kwargs)
     return deco
 
+def showDevIcons(func):
+    @wraps(func)
+    def deco(*args, **kwargs):
+        if lfEval("get(g:, 'Lf_ShowDevIcons', 0)") == "1":
+            content = func(*args, **kwargs)
+            if isinstance(content, AsyncExecutor.Result):
+                return content
+            else:
+                # In case of Windows, line feeds may be included when reading from the cache.
```

Comment:
This place is executed every time `getContent()` is called, it is too slow if the length of content is very large.
I think we can do in this way:
1. remove this decorator from `getContent()`
2. add the decorator for `_getFilesFromCache()` and `_getFileList()`
3. the first return of `getContent()` should be considered.
4. `if isinstance(content, AsyncExecutor.Result):
                return content` can be removed from this decorator, no need to check `if isStartDevIcons(line)` because no icon from the return value of `_getFilesFromCache()` and `_getFileList()`.
5. this decorator can also be removed from `getFreshContent()`

Reply:
Thanks!
I fixed c2c15c6f8dc76f8967b4f283dd9e05c64d6c3444.
Also, I added `@removeDevIcons` to `_writeCache()`.
This is to make sure that the cache file does not contain any icons.